lib/Dialect/HW/HWOps.cpp

```diff
@@ -979,15 +979,15 @@ static void print(OpAsmPrinter &printer, hw::StructCreateOp op) {
 static ParseResult parseStructExplodeOp(OpAsmParser &parser,
                                         OperationState &result) {
   OpAsmParser::OperandType operand;
-  StructType declType;
+  TypeAliasOr<StructType> declType;
```

Comment:
I didn't notice the in the Array PR... Does ODS generate this C++ constraint? If so, neat!

Reply:
Oh.. Just saw that addition in the array pr. Nevermind.